### DISCUSSION TITLE
gpu cannot use in mmdet

### DISCUSSION THREAD

**Discussion by GolvenDong** [obss/sahi#754] [Q&A] [answered]
I can't use gpu when run sahi with mmdet，

`detection_model = AutoDetectionModel.from_pretrained(
    model_type='mmdet',
    model_path=model_path,
    config_path=config_path,
    confidence_threshold=0.4,
    image_size=640,
    device="cuda:0", # or 'cuda:0'
)`

the error message is :No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda',while my other projects can run well on cuda.Is there anyone can solve this problem?

**Comment by fcakyon** [comment]
@GolvenDong thanks a lot for bringing up this bug, it was introduced in `sahi==0.11.4`!

This PR will fix it: https://github.com/obss/sahi/pull/756

**Comment by fcakyon** [comment, marked as answer]
It is now fixed in `sahi==0.11.5`: https://github.com/obss/sahi/discussions/757

**Reply by GolvenDong** [reply]
Thanks a lot,I have tried in sahi==0.11.5 successfully~~
